crates/humanode-peer/src/cli/run.rs

```diff
@@ -79,14 +79,17 @@ pub async fn run() -> sc_cli::Result<()> {
                         task_manager,
                         import_queue,
                         ..
-                    } = service::new_partial(&config)?;
+                    } = service::new_partial(&config).await?;
                     Ok((cmd.run(client, import_queue), task_manager))
                 })
                 .await
         }
         Some(Subcommand::PurgeChain(cmd)) => {
             let runner = root.create_humanode_runner(cmd)?;
-            runner.sync_run(|config| cmd.run(config.substrate.database))
+            runner.sync_run(|config| {
+                remove_frontier_offchain_db(cmd, &config)?;
+                cmd.run(config.substrate.database)
```

Comment:
Is it safe to do it like this? Need to check the substrate command code here.

Reply:
Previously we run `cmd.run(config.substrate.database)` just for substrate related data only. Now we include deleting frontier offchain db before the base command depending on frontier db type. 

The implementation is based on reviewing moonbeam and frontier codes:
- https://github.com/moonbeam-foundation/moonbeam/blob/7879bf787f8a279335bab656606f8f2672609dce/node/cli/src/command.rs#L300
- https://github.com/polkadot-evm/frontier/blob/d4aa56a026980a52c0d5fae6e8299ac5a8a0ef53/template/node/src/command.rs#L115